### PR TITLE
[Keyboard]Atlantis AK7

### PR DESCRIPTION
**Pull request by peepeetee**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/atlantis/ak7/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+
+	switch (id) {
+
+	}
+	return MACRO_NONE;
+}
+
+void matrix_init_user(void) {
+}
```
Comment (drashna) [inline keyboards/atlantis/ak7/keymaps/default/keymap.c:26]:
```suggestion
```

File: keyboards/atlantis/ak7/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+	}
+
+	if (usb_led & (1 << USB_LED_KANA)) {
+
+	} else {
+
+	}
+
+}
```
Comment (drashna) [inline keyboards/atlantis/ak7/keymaps/default/keymap.c:149]:
```suggestion
```

and: https://docs.qmk.fm/#/feature_led_indicators?id=configuration-options

File: keyboards/atlantis/ak7/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,149 @@
+#include "ak7.h"
```
Comment (drashna) [inline keyboards/atlantis/ak7/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/atlantis/ak7/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+
+	switch (id) {
+
+	}
+	return MACRO_NONE;
+}
+
+void matrix_init_user(void) {
+}
```
Comment (drashna) [inline keyboards/atlantis/ak7/keymaps/via/keymap.c:26]:
```suggestion
```

File: keyboards/atlantis/ak7/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+	}
+
+	if (usb_led & (1 << USB_LED_KANA)) {
+
+	} else {
+
+	}
+
+}
```
Comment (drashna) [inline keyboards/atlantis/ak7/keymaps/via/keymap.c:149]:
```suggestion
```

File: keyboards/atlantis/ak7/keymaps/via/rules.mk [hunk 6]
```diff
@@ -0,0 +1,8 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+LEADER_ENABLE = no
+FAUXCLICKY_ENABLE = no
+MIDI_ENABLE = no
+BLUETOOTH_ENABLE = no
+KEY_LOCK_ENABLE = no
+TERMINAL_ENABLE = no
```
Comment (drashna) [inline keyboards/atlantis/ak7/keymaps/via/rules.mk:8]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/atlantis/ak7/info.json [hunk 7]
```diff
@@ -0,0 +1,31 @@
+{
+    "keyboard_name": "AK21", 
+    "maintainer": "Atlantis", 
+    "layouts": {
+        "LAYOUT_numpad_6x4": {
```
Comment (zvecr) [inline keyboards/atlantis/ak7/info.json:5]: Does not match the contents of `keyboards/atlantis/ak7/ak7.h`

**Review by zvecr — CHANGES_REQUESTED** [review]
Please ensure that submissions actually compile

File: keyboards/atlantis/ak7/config.h [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+#define RGBLIGHT_LAYERS_OVERRIDE_RGB_OFF //RGB LAYERS在RGB关闭的情况下仍然有效，实现全局分层指示灯。
+#define RGBLED_NUM 30
+#define RGBLIGHT_HUE_STEP 8
+#define RGBLIGHT_SAT_STEP 8
+#define RGBLIGHT_VAL_STEP 8
+#define RGBLIGHT_SLEEP
+#define RGBLIGHT_LIMIT_VAL 200
+#endif
+
+#endif
```
Comment (zvecr) [inline keyboards/atlantis/ak7/config.h:71]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/atlantis/ak7/rules.mk [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+
+# Build Options
+#   change yes to no to disable
+#
+
```
Comment (fauxpark) [inline keyboards/atlantis/ak7/rules.mk:11]:
```suggestion
```

File: keyboards/atlantis/ak7/rules.mk [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+AUDIO_ENABLE = no           # Audio output
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
```
Comment (fauxpark) [inline keyboards/atlantis/ak7/rules.mk:20]:
```suggestion
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/atlantis/ak7/rules.mk [hunk 11]
```diff
@@ -0,0 +1,21 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+
```
Comment (fauxpark) [inline keyboards/atlantis/ak7/rules.mk:7]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/atlantis/ak7/readme.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+
+    make atlantis/ak7:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
```
Comment (noroadsleft) [inline keyboards/atlantis/ak7/readme.md:25]: `(0,0)` isn't a keyswitch position on this board; we should change it to `(1,0)`

```suggestion
* **Bootmagic reset**: Hold down the key at (1,0) in the matrix (the top left key) and plug in the keyboard
```

File: keyboards/atlantis/ak7/config.h [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+#define RGBLIGHT_LAYERS
+#define RGBLIGHT_LAYER_BLINK
+#define RGBLIGHT_LAYERS_OVERRIDE_RGB_OFF //RGB LAYERS在RGB关闭的情况下仍然有效，实现全局分层指示灯。
+#define RGBLED_NUM 30
+#define RGBLIGHT_HUE_STEP 8
+#define RGBLIGHT_SAT_STEP 8
+#define RGBLIGHT_VAL_STEP 8
+#define RGBLIGHT_SLEEP
+#define RGBLIGHT_LIMIT_VAL 200
+#endif
```
Comment (noroadsleft) [inline keyboards/atlantis/ak7/config.h:69]:
```suggestion
#endif

/* Bootmagic Lite key configuration */
#define BOOTMAGIC_LITE_ROW 1
#define BOOTMAGIC_LITE_COLUMN 0
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: keyboards/atlantis/ak7/info.json [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+            "layout": [
+                {"x":0, "y":0},
+                {"x":1, "y":0},
+                {"x":2, "y":0},
+                {"x":0, "y":1},
+                {"x":1, "y":1},
+                {"x":2, "y":1},
+                {"x":0, "y":2},
+                {"x":1, "y":2},
+                {"x":2, "y":2},
```
Comment (noroadsleft) [inline keyboards/atlantis/ak7/info.json:15]: Syntax error here.

```suggestion
                {"x":2, "y":2}
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]
Looks good to me.

File: keyboards/atlantis/ak7/keymaps/default/keymap.c [hunk 15]
```diff
@@ -0,0 +1,108 @@
...
+		KC_F1, KC_F2, KC_RSFT),
+
+	[1] = LAYOUT(
+		KC_A, KC_TRNS, KC_B,
+		RGB_TOG, RGB_MOD, KC_TRNS,
+		KC_TRNS,KC_TRNS,KC_TRNS)
+};
+
+
+bool is_alt_tab_active = false; // ALT_TAB激活状态。
```
Comment (fauxpark) [inline keyboards/atlantis/ak7/keymaps/default/keymap.c:20]: Could these comments be translated to English?

Comment (peepeetee) [reply keyboards/atlantis/ak7/keymaps/default/keymap.c:20]: From what I can see the variable names are pretty self evident, and the comments are just for what the variables do

Comment (fauxpark) [reply keyboards/atlantis/ak7/keymaps/default/keymap.c:20]: Then perhaps the comments aren't needed at all.

**Comment by resuna** [comment]
Why was this never approved? It seems like all it needed was transalting "激活状态" to "activated".

**Pull request closed without merging**